esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -309,13 +311,16 @@
     ``plot_func``. String arguments can include facets in curly brackets which
     will be derived from the corresponding dataset, e.g., ``{project}``,
     ``{short_name}``, ``{exp}``. Examples: ``default: {levels: 2}, CMIP6:
-    {vmin: 200, vmax: 250}``.
+    {vmin: 200, vmax: 250}``. In addition to the normalization_ options
+    supported by the plot function, the option ``norm=centered``. In this case,
```

Comment:
```suggestion
    supported by the plot function, the option ``norm: centered``. In this case,
```
Also the sentence structure, see above.